examples/workflow-standalone/src/app.ts

```diff
@@ -57,20 +61,39 @@ async function initialize(connectionProvider: MessageConnection): Promise<void>
     });
     await configureServerActions(result, diagramType, container);
 
-    const actionDispatcher = container.get(GLSPActionDispatcher);
+    const actionDispatcher: GLSPActionDispatcher = container.get(GLSPActionDispatcher);
 
     await client.initializeClientSession({ clientSessionId: diagramServer.clientId, diagramType });
+
     actionDispatcher.dispatch(
         RequestModelAction.create({
             options: {
                 sourceUri: `file://${examplePath}`,
-                diagramType
+                diagramType,
+                isReconnecting
             }
         })
     );
     actionDispatcher.dispatch(RequestTypeHintsAction.create());
     await actionDispatcher.onceModelInitialized();
     actionDispatcher.dispatch(EnableToolPaletteAction.create());
+
+    if (isReconnecting) {
+        const message = `Connection to the ${id} glsp server got closed. Connection was successfully re-established.`;
```

Comment:
Offtopic: Dispatching `ServerStatusActions` or `ServerMessageActions` on the client side seems a bit weird.
In general, these actions are just plain status or message notifications and don't have to necessarily come from the server. Maybe we should consider a rename here to a more generic `StatusAction` and `MessageAction` or at the very least make it clear in the documentation that this actions can also be dispatched from the client side.
@planger  WDYT?

Reply:
True, I'm in favor of a rename to `StatusAction` and `MessageAction`. However, I don't consider this a high prio.

Reply:
Me neither, nevertheless we should probably do it as part of the 2.0 Release. It' an API break after all, even if its just a minor one.

Reply:
Sounds good, I'll create an issue for this.

Reply:
See https://github.com/eclipse-glsp/glsp/issues/1071